src/content/tutorials/build-a-slackbot/index.md

```diff
@@ -0,0 +1,696 @@
+
+```sh
+---
+header: Set the SLACK_WEBHOOK_URL secret
+---
+$ wrangler secret create SLACK_WEBHOOK_URL
+Enter the secret text you'd like assigned to the variable name on the script named slack-bot-ENVIRONMENT_NAME: https://hooks.slack.com/services/abc123
+```
+
+#### Handling errors
+
+Similarly to the `lookup` function handler, the `webhook` function handler should include some basic error handling. Unlike `lookup`, which sends responses directly back into Slack, if something goes wrong with your webhook, it may be useful to actually generate an erroneous response, and return it to GitHub. To do this, wrap the majority of the `webhook` function handler in a try/catch block, and construct a new response with a status code of 500, and return it. The final version of `src/handlers/webhook.js` looks like this:
+
+```javascript
+---
+filename: src/handlers/webhook.js
+highlight: [4, 22, 23, 24, 25]
+---
+import { constructGhIssueSlackMessage } from '../utils/slack'
+
+export default async request => {
+  try {
+    const body = await request.text()
+    const { action, issue, repository } = JSON.parse(body)
+    const prefix_text = `An issue was ${action}:`
+    const issue_string = `${repository.owner.login}/${repository.name}#${issue.number}`
+    const blocks = constructGhIssueSlackMessage(
+      issue,
+      issue_string,
+      prefix_text,
+    )
+
+    const postToSlack = await fetch(SLACK_WEBHOOK_URL, {
+      body: JSON.stringify({ blocks }),
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+    })
+
+    return new Response('OK')
+  } catch (err) {
+    const errorText = 'Unable to handle webhook'
+    return new Response(errorText, { status: 500 })
+  }
+}
+```
+
+## Publish
+
+And with that, you’re finished writing the code for your Slack bot! Pat yourself on the back — it was a lot of code, but now we can move on to the final steps of this tutorial: actually publishing your application.
+
+Wrangler has built-in support for bundling, uploading, and releasing your Cloudflare Workers application. To do this, we’ll run `wrangler publish`, which will _build_ and _publish_ your code:
+
+![Verify Wrangler Installation](./media/publish.gif)
+
+Publishing your Workers application should now cause issue updates to start appearing in your Slack channel, as the GitHub webhook can now successfully reach your Workers webhook route:
+
+![Create New Issue](./media/create-new-issue.gif)
+
+## Resources
+
+In this tutorial, you built and published a Cloudflare Workers application that can respond to GitHub webhook events, and allow GitHub API lookups within Slack. If you’d like to see the full source code for this application, you can find the repo [on GitHub](https://github.com/signalnerve/workers-slack-bot).
+
+If you want to get started building your own projects, check out the quick-start templates we’ve provided in our [Template Gallery](/templates).
```

Comment:
Update to new ending seen in other new l18n tutorial (Template Gallery): 

https://github.com/cloudflare/workers-docs-engine/blob/5b0395e72f3c4d7d052fda7a4d8fc4d1b3946490/src/content/tutorials/localize-a-website/index.md#L281